src/pay/batch_payer.py

```diff
@@ -325,33 +359,46 @@ def pay(self, payment_items_in, dry_run=None):
                 )
             )
 
-            payment_logs.extend(payments_log)
+            for payment_item in payment_items_chunk:
+                if (
+                    payment_item.paid == PaymentStatus.PAID
```

Comment:
Why do we need here again a check for the payment status? As I am understanding the payment_items_chunk only consists of paid, done and injected payments ^^

Reply:
I removed the if check so that the regression tests pass. Please let me know if it's fine and what you had in mind implementing it.